snap/hooks/configure

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+import sys
+
+import snaphelpers
+
+
+def read_config_key(snap_config: snaphelpers.SnapConfig, key: str, default=None):
+    try:
+        return snap_config.get(key)
+    except snaphelpers._conf.UnknownConfigKey:
+        return default
+
+
+def validate_provider(snap_config: snaphelpers.SnapConfig):
+    provider = read_config_key(snap_config, "provider")
+
+    if provider and provider not in ["lxd", "multipass"]:
```

Comment:
multipass should only be valid once we add support for it

Reply:
I put it in initially so we can toggle it once I start adding multipass support.  I expect this option would not be documented as supported until it is complete.